README.md

```diff
@@ -1 +1,160 @@
-# Rotations
+# Rotations.jl
+
+This package implements various rotation parameterizations and defines conversions between them.  All rotation variables are stored as as immutable types.
+
+This package assumes [active (right handed) rotations](https://en.wikipedia.org/wiki/Active_and_passive_transformation) wheere applicable.
```

Comment:
Ah, I've been thinking of it as R_passive = inv(R_active) which matches
inverting theta in the AngleAxis parametrization.

On 11 May 2016 at 10:13, Chris Foster <notifications@github.com> wrote:

> In README.md
> <https://github.com/FugroRoames/Rotations.jl/pull/2#discussion_r62775242>:
>
> > @@ -1 +1,160 @@
> > -# Rotations
> > +# Rotations.jl
> > +
> > +This package implements various rotation parameterizations and defines conversions between them.  All rotation variables are stored as as immutable types.
> > +
> > +This package assumes [active (right handed) rotations](https://en.wikipedia.org/wiki/Active_and_passive_transformation) wheere applicable.
>
> No, right handed is correct. I was just wondering why it was placed in
> brackets - the way it's written, it looks like active=>right handed. Those
> are conceptually distinct though I think?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/FugroRoames/Rotations.jl/pull/2/files/fb3a7882257b93afed97cfaa9bf56669bbf410dc#r62775242>
>